integration/ct_integration_local.cfg

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Putting the paths inside the test file makes it clear that this config data is test only. 
Just a bit of self documentation, a speedup (avoids a file.Open) and compile time checking of the data structure.